apps/elekto/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: elekto-config
```

Comment:
```suggestion
metadata:
  name: elekto-config
  namespace: elekto
```

Reply:
ooops, does namespace need to be added to all the other files as well?

Reply:
Yes for all the files deploying Kubernetes resources. Otherwise the resources will be deployed to the default namespace.